pkg/webui/console/views/device-add/repository/hints/other-hint.js

```diff
@@ -0,0 +1,67 @@
+// Copyright © 2021 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import { defineMessages } from 'react-intl'
+
+import Link from '@ttn-lw/components/link'
+
+import Message from '@ttn-lw/lib/components/message'
+
+import PropTypes from '@ttn-lw/lib/prop-types'
+
+import style from './hints.styl'
+
+const m = defineMessages({
+  hintTitle: 'Your end device will be added soon!',
+  hintMessage:
+    "We're sorry, but your device is not yet part of The Things Device Repository. You can use <ManualLink>manual device registration</ManualLink>, using the information your end device manufacturer provided e.g. in the product's data sheet. You can find more assistance on manual device creation <GuideLink>here</GuideLink>.",
```

Comment:
```suggestion
    "We're sorry, but your device is not yet part of The Things Device Repository. You can use <ManualLink>manual device registration</ManualLink>, using the information your end device manufacturer provided e.g. in the product's data sheet. Please also refer to hour documentation on <GuideLink>Adding Devices</GuideLink>.",
```

Sorry, this needs to reword a little (in deviation of the wireframe wording).